src/PhenX.EntityFrameworkCore.BulkInsert/Metadata/PropertyMetadata.cs

```diff
@@ -15,6 +15,8 @@ internal sealed class PropertyMetadata(IProperty property,  SqlDialectBuilder di
 
     public string QuotedColumName { get; } = dialect.Quote(property.GetColumnName());
 
+    public string StoreDefinition { get; } = GetStoreDefinition(property);
```

Comment:
where is it used ?

Reply:
Not anymore. I tested some stuff with SQL Server and whether it makes more sense to create the temp table manually.

The point is that the identity constraints are somehow copied to the SQL table, therefore you cannot make upserts when you have a auto-incremented ID. I still want to continue experimenting with that.

Reply:
It is used now. I create the temp table manually for SQL server now, therefore you can make upserts with auto-incremented IDs now.